CommandLine/libEGM/CMakeLists.txt

```diff
@@ -36,14 +46,23 @@ target_link_libraries(${LIB} PRIVATE pugixml)
 
 find_package(RapidJSON REQUIRED)
 include_directories(${LIB} PRIVATE ${RAPIDJSON_INCLUDE_DIRS})
-target_link_libraries(${LIB} PRIVATE rapidjson)
+if(MSVC)
+  target_link_libraries(${LIB} PRIVATE rapidjson)
+endif(MSVC)
```

Comment:
What the hell is this? The YYP reader requires rapidjson no matter what, even on Linux.